util/versions.py

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env python
 
+from string import Template
+import argparse
 import json
-import logging as log
 import os
 import sys
 
-log.basicConfig(level=log.INFO, format="%(levelname)s: %(message)s")
-
-
 def key(v):
     if v == "master":
-        return float("inf")
-    if v == "stable":
         return sys.maxsize
-    if v == "beta":
+    if v == "stable":
         return sys.maxsize - 1
+    if v == "beta":
+        return sys.maxsize - 2
     if v == "pre-1.29.0":
         return -1
+    if not v.startswith("rust-"):
+        return None
```

Comment:
This removes the old versions from the index, right? I think this is a good step. And with that, I will probably follow up with removing all of them from the `gh-pages` branch. But will wait a few weeks to see if someone complains.

Reply:
Oh right. I think I deleted those `pre-1.29.0` dirs a while ago already.